CMakeLists.txt

```diff
@@ -2,6 +2,7 @@ cmake_minimum_required(VERSION 3.1 FATAL_ERROR)
 project(norse)
 
 find_package(Torch REQUIRED)
-find_package (Python3 REQUIRED COMPONENTS Development)
+FIND_PACKAGE(PythonInterp 3 REQUIRED)
```

Comment:
These are both deprecated in favour of what was used before. Perhaps you are using an outdated version of CMake?

Reply:
Likely. I tried again, and it seemed to work. Strange.